modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase

```diff
@@ -60,9 +60,13 @@ definition {
 		}
 
 		task ("Then: There is no hyphen in any header text") {
-			AssertTextNotPresent(
-				locator1 = "FrontendDataSet#HEADER_ROW",
-				value1 = "-");
+			var header = selenium.getText("FrontendDataSet#HEADER_ROW");
+
+			var containsHyphen = StringUtil.contains("${header}", "-");
+
+			TestUtils.assertEquals(
+				actual = "${containsHyphen}",
+				expected = "false");
```

Comment:
This is very interesting. Does this work better to capture all the text in the row than the function AssertTextNotPresent?

Reply:
Yes, its a workaround for the error related to the assertion of hyphen.